docs/api-testing/schedule-test.md

```diff
@@ -18,19 +18,22 @@ The **Schedule** tool, accessible from within each test, allows you to schedule
 ## Creating a New Test Schedule
 
 :::caution Required Step
-Do not proceed until you've [published the working copy of your test](/api-testing/quickstart/#publish-your-test).
+Schedule is available only for [published tests](/api-testing/quickstart/#publish-your-test).
 :::
 
-1. Click the **Schedule** tab to access the Scheduler.<br/><img src={useBaseUrl('img/api-fortress/2021/02/scheduleTab.png')} alt="Schedule Tab"/>
-2. Click **Create Schedule**.<br/><img src={useBaseUrl('img/api-fortress/2021/02/createSchedule.png')} alt="Create Schedule" width="450"/>
+1. From within the **Composer**, click the **Schedule** tab to access the Scheduler.<br/><img src={useBaseUrl('img/api-fortress/2021/02/scheduleTab.png')} alt="Schedule Tab"/>
```

Comment:
@LoryApiFortress , I think we may have discussed before -- we should be using **Compose** and **Schedule** (not  **Composer** and **Scheduler**) to match the UI elements. 

You could also speak with the Growth Team to propose changing the UI elements to **Composer** and **Scheduler**.